### PR TITLE
Assurance docs for mobile platforms + release notes updates

### PR DESCRIPTION
**Pull request by PravinPK**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -12,6 +12,37 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
...
+
+#### Flutter Assurance 1.0.1
+
+- Assurance extension released for Flutter. See: [Assurance Flutter](https://pub.dev/packages/flutter_assurance)
+
+#### Xamarin Assurance 0.0.1
+
+- Assurance extension released for Xamarin iOS. See : [Xamarin Assurance iOS](https://www.nuget.org/packages/Adobe.AEPAssurance.iOS/)
+- Assurance extension released for Xamarin Android. See : [Xamarin  Assurance Android](https://www.nuget.org/packages/Adobe.AEPAssurance.Android/) 
+
```
Comment (ahmyr) [inline release-notes.md:37]: To stay consistent with the naming for the other platforms, can we put "Assurance" first... Assurance Xamarin iOS and Assurance Xamarin Android.

File: release-notes.md [hunk 2]
```diff
@@ -12,6 +12,37 @@ The Adobe Experience Platform Mobile SDK will soon be available in Swift as an i
...
+#### Cordova Assurance 0.0.1
+
+- Support for Assurance extension in Cordova. See : [Assurance Cordova](https://www.npmjs.com/package/@adobe/cordova-aepassurance)
```
Comment (ahmyr) [inline release-notes.md:40]: To stay consistent with the others, can we change this to "Assurance extension released for Cordova".

File: using-mobile-extensions/adobe-experience-platform-assurance/README.md [hunk 3]
```diff
@@ -74,39 +74,119 @@ import AEPAssurance // <-- import the AEPAssurance library
...
+
 {% endtab %}
 
 {% tab title="Unity" %}
 #### C\#
 
-{% hint style="danger" %}
-Updated support for Unity is coming. This section will be updated shortly.
-{% endhint %}
+1. After importing the [AEPAssurance.unitypackage](https://github.com/adobe/unity-aepassurance/tree/master/bin), the Assurance extension for Unity can be added with following code in the MainScript
```
Comment (ahmyr) [inline using-mobile-extensions/adobe-experience-platform-assurance/README.md:161]: Add a "the" for "added with the following code".
And, add a : after "MainScript".

File: using-mobile-extensions/adobe-experience-platform-assurance/README.md [hunk 4]
```diff
@@ -74,39 +74,119 @@ import AEPAssurance // <-- import the AEPAssurance library
...
+
 {% endtab %}
 
 {% tab title="Xamarin" %}
 #### C\#
 
-{% hint style="danger" %}
-Updated support for Xamarin is coming. This section will be updated shortly.
-{% endhint %}
+1. After adding the [iOS](https://www.nuget.org/packages/Adobe.AEPAssurance.iOS/) or [Android](https://www.nuget.org/packages/Adobe.AEPAssurance.Android/) AEPAssurance NuGet package, the Assurance extension can be added by this import statement
```
Comment (ahmyr) [inline using-mobile-extensions/adobe-experience-platform-assurance/README.md:178]: Add a : at the end of this sentence for consistency.

File: using-mobile-extensions/adobe-experience-platform-assurance/README.md [hunk 5]
```diff
@@ -167,35 +247,105 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% endtab %}
 
 {% tab title="React Native" %}
-{% hint style="danger" %}
-Updated support for React Native is coming. This section will be updated shortly.
-{% endhint %}
+
+### JavaScript
+
+When using React Native, registering AEPAssurance with Mobile Core should be done in native code which is shown under the Android and iOS tabs.
```
Comment (ahmyr) [inline using-mobile-extensions/adobe-experience-platform-assurance/README.md:253]: Change to:

When using React Native, register AEP Assurance with Mobile Core in native code as shown on the Android and iOS tabs. (and add a link).

Make same change for other platforms below.

File: using-mobile-extensions/adobe-experience-platform-assurance/adobe-experience-platform-assurance-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# Release Notes
+
```
Comment (ahmyr) [inline using-mobile-extensions/adobe-experience-platform-assurance/adobe-experience-platform-assurance-release-notes.md:2]: Make same changes here as listed above for main release notes.

**Review by praschetan — APPROVED** [review]

**Pull request merged**